content.md

```diff
@@ -67,6 +75,7 @@ A collection of tools and resources to enhance your gameplay!
 - [score inspector](https://github.com/darkchii/score-inspector-extension) - A userscript that adds more stats to user profiles on the official osu! website (Extension compatible with the [osu! scores inspector](https://score.kirino.sh/))
 - [osu! preview](https://github.com/JerryZhu99/osu-preview) - Preview a beatmap without downloading it
 - [osu! subdivide nations](https://github.com/Cavitedev/osu-subdivide-nations) - Display the regional flags and names of players on the official osu! website as well as other unofficial sites (Extension compatible with [osu! World](https://osuworld.octo.moe/))
+- [ezpp!](https://github.com/cl8n/awesome-osu?tab=readme-ov-file#installing) - Calculate how much PP a map is worth for a particular score
```

Comment:
incorrect url -> https://github.com/oamaok/ezpp, mention it's a browser extension

Reply:
The repository weirdly decided to link the chrome/firefox addon marketplaces at the bottom of the readme, so I linked directly to the installing section of the page. It's in the browser extensions category, I don't think that needs another mention.

Reply:
But the link you placed *doesn't go anywhere*.

Best solution is to just link the browser extensions directly.

Reply:
what is bro on about none of that is true

Reply:
Oh, right, I don't know how that happened

Reply:
Should be good now

Reply:
Recent commit induced a CI failure.

Reply:
Yeah, I the chrome link was redirecting, I just copied the links from readme without checking

Reply:
please just only link the github, none of the other ones have the platform specific urls and it shouldn't either as I think that should be left up to the maintainer of the project on github

Reply:
Alright, removed the direct links